myblog/blog/api/serializers.py

```diff
@@ -0,0 +1,8 @@
+from ..models import BlogPost
+from rest_framework import serializers
+
+
+class BlogPostSerializer(serializers.Serializer):
+    body = serializers.CharField()
+    text = serializers.CharField(source = 'title')
+    user = serializers.IntegerField(source = 'author.id')
```

Comment:
Using `serializers.ModelSerializer ` is cleaner when you work with typical Django Models, as they already declare the field types for you. Let's try refactoring this serializer into a ModelSerializer, remember to check that your tests are passing after that refactor.

Reply:
I use this to change the name in my serializer to the requests names in the exercise, I've done the change based in https://stackoverflow.com/a/43248297